lib/src/arkit_reference_node.dart

```diff
@@ -30,9 +32,11 @@ class ARKitReferenceNode extends ARKitNode {
   /// Defaults to path from Main Bundle
   /// If path from main bundle fails, will search as full file path
   final String url;
+  final AssetType assetType;
 
   @override
   Map<String, dynamic> toMap() => <String, dynamic>{
         'url': url,
+        'assetType': assetType.name
```

Comment:
Please use `assetType.index` for consistency. In that case, you would not need to bump the minimum SDK version.

Reply:
Edited.